pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1322,26 +1313,24 @@ func getMatchingTopologyPairsOfExistingPod(newPod *v1.Pod, existingPod *v1.Pod,
 	}
 	return topologyMaps, nil
 }
-func (c *PodAffinityChecker) getMatchingAntiAffinityTopologyPairs(pod *v1.Pod, allPods []*v1.Pod) (*topologyPairsMaps, error) {
+
+func (c *PodAffinityChecker) calculateMatchingAntiAffinityTopologyPairs(pod *v1.Pod, allPods []*v1.Pod) (*topologyPairsMaps, error) {
```

Comment:
To be consistent, please rename this to `getMatchingAntiAffinityTopologyPairsOfAllPods`.

Reply:
done.